in_toto/gpg/common.py

```diff
@@ -0,0 +1,151 @@
+"""
+<Module Name>
+  gpg/common.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 15, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  signature-algorithm-agnostic frontend for the gpg_* parsing, signing and
+  verifying functions. These functions select the appropriate functions for
+  each algorithm and call it.
+"""
+import struct
+import binascii
+
+import in_toto.log
+from in_toto.gpg.constants import (PACKET_TYPES,
+        SUPPORTED_PUBKEY_PACKET_VERSIONS, SIGNATURE_TYPE_CANONICAL,
+        SUPPORTED_SIGNATURE_PACKET_VERSIONS, SUPPORTED_SIGNATURE_ALGORITHMS,
+        SUPPORTED_HASH_ALGORITHMS, SIGNATURE_HANDLERS, FULL_KEYID_SUBPACKET,
+        FULLY_SUPPORTED_MIN_VERSION)
+
+from in_toto.gpg.util import (compute_keyid, parse_packet_header,
+    parse_subpackets, get_version)
+import in_toto.gpg
+
+
+
+# XXX this doesn't support armored pubkey packets, so use with care.
```

Comment:
How about linking to a page that explains armored pubkey packets?  Is the [OpenPGP Message Format RFC](https://tools.ietf.org/html/rfc4880#section-6.2) a good one?